lessons/operator/02/README.md

```diff
@@ -1,257 +1,267 @@
+
+   To install `docker-compose`, please read the ["Install Docker Compose"](https://docs.docker.com/compose/install/) documentation.
+
+1. **Git Client (`git`)**
+
+   To install `git`, please read the [`git` downloads](https://git-scm.com/downloads) page.
+   
+1. **Supported Web Browser**
+
+   The workshop expects that you have a modern web browser installed. 
+   Please verify that you have a recent version of one of these common browsers:
+   - [Chrome](https://www.google.com/chrome/fast-and-secure/)
+   - [Safari](https://support.apple.com/en-us/HT204416)
+   - [Firefox](https://www.mozilla.org/en-US/firefox/new/)
+   - [Microsoft Edge](https://www.microsoft.com/en-us/edge)
+   
+1. **Compatible Text Editor**
+
+   This workshop involves creating and editing plain-text configuration files and monitoring code templates.
+   We suggest using a text editor such as:
+   - [Microsoft Visual Studio Code](https://visualstudio.microsoft.com)
+   - [Atom](https://atom.io)
+   - [Sublime Text](https://www.sublimetext.com)
+   - [`vim`](https://www.vim.org/download.php)
+   - [`emacs`](https://www.gnu.org/software/emacs/download.html)
+   
+   _NOTE: Windows users should avoid using `Notepad.exe` which can save edited files in incompatible encodings._
+   
+1. **Optional CLI Tools**
+
+   The workshop includes examples using some optional CLI utilties:
+   - `jq` ([website](https://stedolan.github.io/jq/), [downloads](https://stedolan.github.io/jq/download/))
+
+
+## The Workshop Environment
+
+To fully demonstrate Sensu's capabilities, it is necessary to have a multi-node environment.
+This workshop includes a set of containerized components that represent a simplified Sensu-enabled monitoring environment.
+
+<!-- TODO:  Diagram of Workshop Environment -->
+[![](https://via.placeholder.com/800x250.png?text=TODO:+Insert+Workshop+Diagram+Here+(Issue+62))](https://github.com/sensu/sensu-go-workshop/issues/62)
+
+### The Docker Configuration
+
+We are using `docker-compose` to automate setting up the local workshop environment.
+The containers are configured by a variety of environment variables, defined in the `.env` file included in the [workshop repo](https://github.com/sensu/sensu-go-workshop). 
+
+
+The `docker-compose-default.yaml` defines the configuration of the following containers:
+
+   - Sensu components:
+     - `sensu-backend`: A [Sensu backend](https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-schedule/backend/), [API](https://docs.sensu.io/sensu-go/latest/api/), and [dashboard](https://docs.sensu.io/sensu-go/latest/web-ui/)
+     - `sensu-agent`: A Sensu [agent](https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-schedule/agent/) node
+   - A chat-ops environment, for alerting:
+     - `rocketchat`: [Rocket.Chat](https://rocket.chat/), a web-based chat app similar to Slack
+     - `mongo`: [MongoDB](https://www.mongodb.com/), Rocket.Chat's database
+   - A telemetry stack:
```

Comment:
TBD if we can yank this before the workshop on Tuesday... it would be great if the new docker-compose-default.yaml template was based on Sumo Logic (and ditched InfluxDB + Grafana).